### ISSUE TITLE
5.3 builds breakage on Ubuntu

### ISSUE THREAD

**Issue by MaxDesiatov** [swiftwasm/swift#1962]
I imagine something changed on Ubuntu GitHub runners, as on 20.04 the build fails now with this error:

```
/home/runner/work/swift/indexstore-db/lib/Index/include/IndexStoreDB/Database/IDCode.h:22:3: error: unknown type name 'uint64_t'
  uint64_t Code{};
  ^
/home/runner/work/swift/indexstore-db/lib/Index/include/IndexStoreDB/Database/IDCode.h:23:19: error: unknown type name 'uint64_t'
  explicit IDCode(uint64_t code) : Code(code) {}
                  ^
/home/runner/work/swift/indexstore-db/lib/Index/include/IndexStoreDB/Database/IDCode.h:28:27: error: unknown type name 'uint64_t'
  static IDCode fromValue(uint64_t code) {
                          ^
/home/runner/work/swift/indexstore-db/lib/Index/include/IndexStoreDB/Database/IDCode.h:32:3: error: unknown type name 'uint64_t'
  uint64_t value() const { return Code; }
```

and on 18.04:

```
FAILED: lib/Sema/CMakeFiles/swiftSema.dir/MiscDiagnostics.cpp.o 
/usr/local/bin/sccache /usr/bin/clang++  -DCMARK_STATIC_DEFINE -DGTEST_HAS_RTTI=0 -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -Ilib/Sema -I/home/runner/work/swift/swift/lib/Sema -Iinclude -I/home/runner/work/swift/swift/include -I/home/runner/work/swift/llvm-project/llvm/include -I/home/runner/work/swift/host-build/Ninja-Release/llvm-linux-x86_64/include -I/home/runner/work/swift/llvm-project/clang/include -I/home/runner/work/swift/host-build/Ninja-Release/llvm-linux-x86_64/tools/clang/include -I/home/runner/work/swift/cmark/src -I/home/runner/work/swift/host-build/Ninja-Release/cmark-linux-x86_64/src -Wno-unknown-warning-option -Werror=unguarded-availability-new -fno-stack-protector -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -Wimplicit-fallthrough -Wcovered-switch-default -Wno-class-memaccess -Wno-noexcept-type -Wnon-virtual-dtor -Wdelete-non-virtual-dtor -Wstring-conversion -fdiagnostics-color -ffunction-sections -fdata-sections -Werror=switch -Wdocumentation -Wimplicit-fallthrough -Wunreachable-code -Woverloaded-virtual -DOBJC_OLD_DISPATCH_PROTOTYPES=0 -O3 -DNDEBUG    -fno-exceptions -fno-rtti -Werror=gnu -target x86_64-unknown-linux-gnu -O2 -g0 -DNDEBUG -march=core2 -std=c++14 -MD -MT lib/Sema/CMakeFiles/swiftSema.dir/MiscDiagnostics.cpp.o -MF lib/Sema/CMakeFiles/swiftSema.dir/MiscDiagnostics.cpp.o.d -o lib/Sema/CMakeFiles/swiftSema.dir/MiscDiagnostics.cpp.o -c /home/runner/work/swift/swift/lib/Sema/MiscDiagnostics.cpp
#0 0x00007f7322e500ea llvm::sys::PrintStackTrace(llvm::raw_ostream&) (/usr/lib/llvm-6.0/bin/../lib/libLLVM-6.0.so.1+0x81e0ea)
#1 0x00007f7322e4e366 llvm::sys::RunSignalHandlers() (/usr/lib/llvm-6.0/bin/../lib/libLLVM-6.0.so.1+0x81c366)
#2 0x00007f7322e4e49b (/usr/lib/llvm-6.0/bin/../lib/libLLVM-6.0.so.1+0x81c49b)
#3 0x00007f73260e08a0 __restore_rt (/lib/x86_64-linux-gnu/libpthread.so.0+0x128a0)
#4 0x00007f7322f6e820 llvm::Value::getContext() const (/usr/lib/llvm-6.0/bin/../lib/libLLVM-6.0.so.1+0x93c820)
#5 0x00007f7322f08f26 llvm::StoreInst::StoreInst(llvm::Value*, llvm::Value*, bool, unsigned int, llvm::AtomicOrdering, unsigned char, llvm::Instruction*) (/usr/lib/llvm-6.0/bin/../lib/libLLVM-6.0.so.1+0x8d6f26)
#6 0x00007f7322f08fd3 llvm::StoreInst::StoreInst(llvm::Value*, llvm::Value*, bool, unsigned int, llvm::Instruction*) (/usr/lib/llvm-6.0/bin/../lib/libLLVM-6.0.so.1+0x8d6fd3)
#7 0x0000564e1277c3ba clang::CodeGen::CodeGenFunction::EmitStoreOfScalar(llvm::Value*, clang::CodeGen::Address, bool, clang::QualType, clang::CodeGen::LValueBaseInfo, clang::CodeGen::TBAAAccessInfo, bool, bool) (/usr/lib/llvm-6.0/bin/clang+0x6713ba)
#8 0x0000564e12789550 clang::CodeGen::CodeGenFunction::EmitStoreThroughLValue(clang::CodeGen::RValue, clang::CodeGen::LValue, bool) (/usr/lib/llvm-6.0/bin/clang+0x67e550)
#9 0x0000564e1275c686 clang::CodeGen::CodeGenFunction::EmitScalarInit(clang::Expr const*, clang::ValueDecl const*, clang::CodeGen::LValue, bool) (/usr/lib/llvm-6.0/bin/clang+0x651686)
#10 0x0000564e1275d019 clang::CodeGen::CodeGenFunction::EmitExprAsInit(clang::Expr const*, clang::ValueDecl const*, clang::CodeGen::LValue, bool) (/usr/lib/llvm-6.0/bin/clang+0x652019)
#11 0x0000564e1273b6ce clang::CodeGen::CodeGenFunction::EmitInitializerForField(clang::FieldDecl*, clang::CodeGen::LValue, clang::Expr*) (/usr/lib/llvm-6.0/bin/clang+0x6306ce)
#12 0x0000564e1279f931 clang::CodeGen::CodeGenFunction::EmitLambdaExpr(clang::LambdaExpr const*, clang::CodeGen::AggValueSlot) (/usr/lib/llvm-6.0/bin/clang+0x694931)
#13 0x0000564e12795eb9 (/usr/lib/llvm-6.0/bin/clang+0x68aeb9)
#14 0x0000564e12795f63 (/usr/lib/llvm-6.0/bin/clang+0x68af63)
#15 0x0000564e12796bb9 clang::CodeGen::CodeGenFunction::EmitAggExpr(clang::Expr const*, clang::CodeGen::AggValueSlot) (/usr/lib/llvm-6.0/bin/clang+0x68bbb9)
#16 0x0000564e12790392 (/usr/lib/llvm-6.0/bin/clang+0x685392)
#17 0x0000564e127959a9 (/usr/lib/llvm-6.0/bin/clang+0x68a9a9)
#18 0x0000564e12795e5e (/usr/lib/llvm-6.0/bin/clang+0x68ae5e)
#19 0x0000564e12796bb9 clang::CodeGen::CodeGenFunction::EmitAggExpr(clang::Expr const*, clang::CodeGen::AggValueSlot) (/usr/lib/llvm-6.0/bin/clang+0x68bbb9)
```

The latter references LLVM 6.0 somehow, so maybe something changed in LLVM/clang there? Not sure what could cause the 20.04 breakage though.

**Comment by MaxDesiatov** [comment]
Maybe this is somehow related? https://github.com/actions/virtual-environments/issues/1536

**Comment by MaxDesiatov** [comment]
Closing as resolved in #1949
